utils/manifest-generator/generate-llbuild-manifest

```diff
@@ -0,0 +1,245 @@
+#!/usr/bin/env python3
+
+import sys
+import argparse
+import uuid
+import random
+
+class Command:
+	def __init__(self, name, index, inputList):
```

Comment:
This is a nit, but we use two-space indent rather than tabs generally on LLVM-style projects.